### PR TITLE
5 new tests sabri

### PR DESCRIPTION
**Pull request by smuti**
can you review @bugraokcu

### CODE REVIEW

File: e2e/tests/steps_file.js [hunk 1]
```diff
@@ -54,30 +54,32 @@ module.exports = function() {
...
+        { name: "_ceg.s", value:"o9h4wn" },
+        { name: "_ceg.u", value:"o9h4wn" },
         { name: "_dc_gtm_UA-12366301-29", value: "1" },
         { name: "_dc_gtm_UA-12366301-46", value: "1" },
         { name:"_ga", value:"GA1.4.909841589.1467053654" },
-        { name:"_ga", value:"GA1.4.909841589.1467053654" },
         { name:"_gat_UA-12366301-1", value:"1" },
         { name:"_prmtn_", value:"%7B%22brandIdentifier%22%3A%20%22udemy%22%2C%20%22endTime%22%3A%20%222050-06-11%2014%3A41%3A20%20UTC%22%2C%20%22hasFilter%22%3A%20true%2C%20%22helloBarId%22%3A%202170%2C%20%22id%22%3A%2048250%2C%20%22pmtag%22%3A%20%22kevin-test-promotion%22%2C%20%22state%22%3A%20false%2C%20%22userId%22%3A%201%7D" },
-        { name:"access_token", value:"Jmm6dsfR7VY43K3mecrkGflbi8PE9PWPAxpmBX3c" },
+        { name:"access_token", value:"0dT0VMIVuEm0oczDFwIPwrCOOAfzlcTRH7mxuE7P" },
```
Comment (bugraokcu) [inline e2e/tests/steps_file.js]: maybe you might want to keep `access_token` only, and get rid of the rest as only access_token is needed.

File: e2e/tests/full/wishlist_test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// TODO: add this to generate test helper
```
Comment (bugraokcu) [inline e2e/tests/full/wishlist_test.js]: We should agree on a naming convention. Do test files start with capital case or lower case?
Compare `e2e/tests/full/Profile_test.js` with `wishlist_test.js` ?

File: e2e/tests/full/wishlist_test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	//checks whishlist contains course text
+	I.waitForText(text, 30);
+});
+
+/**
+* Removes a Course from wishlist
+*/
+Scenario('should remove a course from Wishlist', (I, coursePage) => {
+	//Removes from wishlist
+	coursePage.changeWishlistState("Wishlist");
```
Comment (bugraokcu) [inline e2e/tests/full/wishlist_test.js]: can we call this method as `removeFromWishlist` or something similar? Naming can hint what to do if we remove it whereas we need to remember state definitions for `changeWishlistState`

**Comment by bugraokcu** [comment]
A few inline comments, otherwise lgtm @smuti

File: e2e/tests/full/wishlist_test.js (hunk 2 above)
Comment (smuti) [inline e2e/tests/full/wishlist_test.js]: Pascal casing (Capital) should be good.

File: e2e/tests/full/wishlist_test.js (hunk 3 above)
Comment (smuti) [inline e2e/tests/full/wishlist_test.js]: This is an on/off function. I mean if an item is in wishlist then this func can be used for removing it or if an item isn't in wishlist then this func can be used for adding the item into list. 

If we need remove function than we also need add function.  What is your opinion? 

Should we rename the function and add a new func for adding items to wishlist?

File: e2e/tests/steps_file.js (hunk 1 above)
Comment (smuti) [inline e2e/tests/steps_file.js]: Also we need to keep dj_session_id. We dont need the other cookies.

**Comment by smuti** [comment]
Thank you @bugraokcu. I made the changes

**Pull request merged**